### PR TITLE
Feature/core 8000 geo scatter plot dot coordinates

### PR DESCRIPTION
**Pull request by Navas23**
GeoChartScatterPlot returns coordinates on click event

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 29.5.0
```
Comment (fofi) [inline CHANGELOG.md]: Should this change be a minor or a patch?

File: src/elements/GeoChart/GeoChartScatterPlot/GeoChartScatterPlot.js [hunk 2]
```diff
@@ -255,8 +255,10 @@ function renderSingleGroup (group, d3TipInstance, singleGroupOptions, globalOpti
       unclickedStyle(dotToUnclick, previouslyClickedDot)
     }
 
-    clickedStyle(d3.select(this), d)
-    return singleGroupOptions.onDotClick(d, d.index)
+    const clickedDot = d3.select(this)
+    const dotCoordinates = [parseFloat(clickedDot.attr('cx')), parseFloat(clickedDot.attr('cy'))]
```
Comment (fofi) [inline src/elements/GeoChart/GeoChartScatterPlot/GeoChartScatterPlot.js]: Is this the absolute screen value? I think we should add a test regarding this if possible

File: CHANGELOG.md (hunk 1 above)
Comment (Navas23) [reply CHANGELOG.md]: To me (maybe I'm wrong) a minor is used just for bugfixes, that's why I decided to go for a patch.

File: src/elements/GeoChart/GeoChartScatterPlot/GeoChartScatterPlot.js (hunk 2 above)
Comment (Navas23) [reply src/elements/GeoChart/GeoChartScatterPlot/GeoChartScatterPlot.js]: Yes, adding a test make sense even if it should never fail as the dot is already in the DOM

Comment (fofi) [reply src/elements/GeoChart/GeoChartScatterPlot/GeoChartScatterPlot.js]: I mean, I think we should verify that the returned value is the value we expect.
Imagine that we are getting the value inside the graph container but not the absolute one to the entire screen

Comment (Navas23) [reply src/elements/GeoChart/GeoChartScatterPlot/GeoChartScatterPlot.js]: Ok, I didn't get you the first time. The coordinates are relative to the GeoChart component, is the Geoblink app who decides what to do with the coordinates... Would you add a test anyway? (I think is not needed bc it should work 100%)

**Pull request merged**